nats-jetstream-http-connector/README.md

```diff
@@ -7,7 +7,9 @@ The job of the connector is to read messages from the subject in the given strea
 - `TOPIC`: Subject from which messages are read. It is generally of form - `streamname.subjectname`
 - `HTTP_ENDPOINT`: http endpoint to post request
 - `RESPONSE_TOPIC`: Subject to write responses on success response.  It is generally of form - `response_stream_name.response_subject_name` where streamname should be different then input stream. `response_stream_name` is output stream name. `response_subject_name` subject name where output is send.
+- `RESPONSE_STREAM`-stream to which connector will push messages.
 - `ERROR_TOPIC`: Subject to write errors on failure.  It is generally of form - `err_response_stream_name.error_subject_name` where streamname should be different then input stream. `err_response_stream_name` is error stream name. `error_subject_name` subject name where error output is send.
+- `Error_STREAM`- -stream to which connector will push error messages.
```

Comment:
ERROR_STREAM

Reply:
Please correct formatting to match other descriptions.

`ERROR_STREAM`: stream to which connector will push messages post non-successful invocation of the consumer function.